extended/src/main/java/apoc/util/ExtendedUtil.java

```diff
@@ -2,16 +2,110 @@
 
 import static apoc.export.cypher.formatter.CypherFormatterUtils.formatProperties;
 import static apoc.export.cypher.formatter.CypherFormatterUtils.formatToString;
+import static apoc.util.JsonUtil.OBJECT_MAPPER;
 
+import java.io.IOException;
+import java.io.InputStream;
 import java.math.BigInteger;
 import java.time.Duration;
 import java.time.temporal.TemporalAccessor;
+import java.util.List;
 import java.util.Map;
+import java.util.Spliterators;
 import java.util.stream.LongStream;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import apoc.ml.bedrock.BedrockConfig;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.JsonPath;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+import org.apache.http.client.methods.HttpHead;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpOptions;
+import org.apache.http.client.methods.HttpPatch;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.client.methods.HttpRequestBase;
+import org.apache.http.entity.StringEntity;
 import org.neo4j.graphdb.Entity;
 
 public class ExtendedUtil
 {
+
+    /**
+     * Get the {@link HttpRequestBase} from the method name
+     * Similar to <a href="https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/http/apache/request/impl/ApacheHttpRequestFactory.java#L118">aws implementation</a>
+     */
+    public static HttpRequestBase fromMethodName(String method, String uri) {
+        return switch (method) {
+            case HttpHead.METHOD_NAME -> new HttpHead(uri);
+            case HttpGet.METHOD_NAME -> new HttpGet(uri);
+            case HttpDelete.METHOD_NAME -> new HttpDelete(uri);
+            case HttpOptions.METHOD_NAME -> new HttpOptions(uri);
+            case HttpPatch.METHOD_NAME -> new HttpPatch(uri);
+            case HttpPost.METHOD_NAME -> new HttpPost(uri);
+            case HttpPut.METHOD_NAME -> new HttpPut(uri);
+            default -> throw new RuntimeException("Unknown HTTP method name: " + method);
+        };
+    }
+
+    /**
+     * Similar to JsonUtil.loadJson(..) but works e.g. with GET method as well,
```

Comment:
`loadJSON` should also work with get.

we did a lot of work with the security team on http access and load json.
what happens if we pass in file URLs here etc?
not to fix now due to the time constraint but consider to make it work with `loadJson`